### DISCUSSION TITLE
Does this work on a1111 for Mac?

### DISCUSSION THREAD

**Discussion by tehpupu** [glucauze/sd-webui-faceswaplab#59] [Q&A]
As above, not sure about the requirement..not a tech person :(

**Comment by glucauze** [comment]
It should, but there will be no GPU acceleration.

**Reply by tehpupu** [reply]
Actually I didn't managed to get it to work...there were error but not sure how to capture it on MAC ^^;

**Reply by RichEdg** [reply]
I have it working on my mac, I have had some reasonable results. What I am enjoying is that if there is an animal like a dog in the image as well as a person the Faceswap will swap the input image onto the dog! Not an intended result. I am really struggling to understand the number system to select a face if there are multiple faces. No detailed explanation is given!

**Comment by vicento** [comment]
work well on my MacStudio M1
